lib/v2/douban/other/explore.js

```diff
@@ -10,22 +10,37 @@ module.exports = async (ctx) => {
     const data = response.data;
 
     const $ = cheerio.load(data);
-    const list = $('div[data-item_id]');
-    let itemPicUrl;
+    const list = $('div.item');
 
     ctx.state.data = {
         title: '豆瓣-浏览发现',
         link: 'https://www.douban.com/explore',
         item:
             list &&
             list
-                .map((index, item) => {
+                .map((_, item) => {
                     item = $(item);
-                    itemPicUrl = item.find('a.cover').attr('style').replace('background-image:url(', '').replace(')', '');
+
+                    const haveItemPic = item.find('a.cover').attr('style');
+                    const itemPicHtml = haveItemPic
+                        ? `<img src=${
```

Comment:
Please use [art-template](https://docs.rsshub.app/joinus/script-standard.html#template) for rendering HTML https://github.com/DIYgod/RSSHub/discussions/8359